ipc/src/server.rs

```diff
@@ -174,21 +186,11 @@ impl<M: Metadata, S: Middleware<M>> ServerBuilder<M, S> {
 						outgoing_separator.clone(),
 					)
 				).split();
-				let responses = reader.and_then(move |req| {
-					service.call(req).then(move |response| match response {
-						Err(e) => {
-							warn!(target: "ipc", "Error while processing request: {:?}", e);
-							future::ok(None)
-						},
-						Ok(None) => {
-							future::ok(None)
-						},
-						Ok(Some(response_data)) => {
-							trace!(target: "ipc", "Sent response: {}", &response_data);
-							future::ok(Some(response_data))
-						}
-					})
+				let responses = reader
+					.map(move |req| {
+					service.call(req)
 				})
+				.buffer_unordered(1024)
```

Comment:
Would be good if it was configurable.

Reply:
It should be now. By default, I've set it to 5, I really have no clue what's a _good default_ here.